PaddleCV/object_detection/docs/MODEL_ZOO.md

```diff
@@ -0,0 +1,81 @@
+# Model Zoo and Benchmark
+## Environment
+
+- Python 2.7.1
+- PaddlePaddle 1.5
+- CUDA 9.0
+- CUDNN 7.4
+- NCCL 2.1.2
+
+## Common settings
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
```

Comment:
Batch Normalization layers in backbones are replaced by Affine Channel layers

Reply:
Done